### ISSUE TITLE
/find sometimes give the incorrect location when they are inside an interior

### ISSUE THREAD

**Issue by reemyrobby** [TommyB123/redcountyrp-bugs#154]
When you use '/find' on someone who is in an interior, the script will sometimes give the incorrect location.

This usually happens when entering a block (which will say "N/A"), and entering a flat within a block will give another location (e.g "Hilltop Farm") which does not make sense.

**Comment by wtfrazr** [comment]
This is related to the phone system which has been said not to be enhanced to separate interiors and exteriors. With this issue in mind, Tommy should consider that idea now.

**Comment by CaveFM** [comment]
I've tried this with Valero and when he entered a house with a custom interior it said he was at Come-A-Lot instead of Hilltop where he lives. When I entered the interior with him and I did /find again it said he was at Hilltop Farm instead of Come-A-Lot. 

I've tried this with another house at Hilltop and when he went inside it said Red County instead of Hilltop  Farm and when I entered it said Hilltop Farm.
